Blog: Are We Capturing the Ensemble? (#87)
rs-station/rs-station.github.io: _posts/2026-08-03-are-we-capturing-the-ensemble.md

---
layout: post
title: Are We Capturing the Ensemble?

> subtitle: What structural biology experiments preserve, blur, or discard about molecular distributions

Reviewer: You leave some "punch" on the table with the title & subtitle. Here's my suggestion:

```suggestion
title: A Sharp Look at Fuzzy Ensembles
subtitle: What structural biology experiments can reveal – and obscure – about molecular distributions
```

Reviewer: Or: "Sharp thinking regarding fuzzy maps"

author: Minhuan Li, F. Emil Thomasen, Pilar Cossio
contact: "{minhuanli,fthomasen,pcossio}@flatironinstitute.org"
usemathjax: true
---

> ![Two ways an experiment can encode a conformational ensemble]({{ '/assets/posts/2026-08-03-are-we-capturing-the-ensemble/measurement_types.png' | relative_url }}){: .blog-image-hero}

Reviewer: add "left", "right", "top", ...

Author: good point

Two ways in which an experiment can encode a conformational ensemble. Ensemble-averaged measurements combine contributions from many molecules into each observable. In single-particle or single-molecule measurements, each observation arises from an individual molecule, and the ensemble is represented statistically across many observations.
{: .blog-caption-wide}

> Structural biology is increasingly being asked to answer questions about ensembles rather than single structures. But two opposite intuitions can lead us astray. In cryo-electron microscopy (cryo-EM), a fuzzy or weak region of a reconstructed map is sometimes treated as if it were direct evidence of an ensemble: blur becomes dynamics. In the other direction, an experiment such as X-ray diffraction may be dismissed as incapable of constraining an ensemble because its measurements average over many molecules.

Reviewer: I'd spice this up just a bit by adding a sentence that highlights some of the opportunity presented by ensembles:

"This exciting frontier might let us finally understand allostery, how entropy contributes to structure, and how proteins change their shape when they recognize binding partners."

Or something like that.

{: .post-lead}

These intuitions seem contradictory, but they make the same mistake. Both infer what an experiment can tell us about a molecular distribution from the appearance of its final representation, rather than from how the data were generated. A blurred map is not itself a probability distribution. But neither does averaging an observable necessarily collapse a distribution to a single structure.

The more useful question is therefore not whether a dataset "contains an ensemble." It is which differences between molecular distributions remain observable after the measurement process—and after the processing applied to the data. Two distinct ensembles may produce measurably different signals, or they may become indistinguishable once projected through experimental noise, limited observables, nuisance variables, finite sampling, or data reduction. Understanding that distinction has to come before choosing an ensemble-inference method.

To make this question precise, we first need to distinguish the distribution we want to learn from the one the experiment actually sees.

> ### What distribution enters the experiment?

Reviewer: I am not sure how you can "enter" an experiment.

```suggestion
### What distribution does an experiment report?
```

Author: This section is intended for a short discussion about p_src and p_target, so it is truly "what distribution enters the experiment". In the next section we then talk about how p_src was preserved or reported.

Reviewer: @minhuanli my comment purely concerned English usage (the content: understood). I think you need a different verb, as a subject cannot "enter" an experiment.

> Before asking what information the measurement preserves, we should first be clear about what distribution is being measured. Let $$p_{\mathrm{target}}(x)$$ denote the molecular distribution relevant to the scientific question, and $$p_{\mathrm{src}}(x)$$ the distribution actually present in the prepared sample when the measurement is made. Often we hope that $$p_{\mathrm{src}}$$ faithfully represents $$p_{\mathrm{target}}$$, but the two need not be identical.

Reviewer: ```suggestion
Before asking what information the measurement preserves, we should first be clear about what distribution is being measured. Let $$p_{\mathrm{target}}(x)$$ denote a molecular distribution relevant to the scientific question, and $$p_{\mathrm{src}}(x)$$ the distribution actually present in the prepared sample when the measurement is made. Often we hope that $$p_{\mathrm{src}}$$ faithfully represents $$p_{\mathrm{target}}$$, but the two need not be identical.
```

Author: Not sure why this change is required?

Reviewer: Change the definite "**the** molecular distribution relevant to the scientific question" to indefinite: "**a** molecular distribution".

I am doubtful you can logically argue that there is a single distribution "per question" (if that even makes sense). Feel free to go for it and prove me wrong.

I actually might quibble with the implication here that there is some Platonic "ideal distribution" one seeks at all. That idea sounds imprecise. But that is a broader debate I don't think we need to have.

The distinction is especially concrete in cryo-EM, where vitrification is itself a physical process acting on the ensemble. Recent work by Clark et al. examined how rapid cooling can perturb conformational populations and how equilibrium populations might subsequently be inferred from the vitrified ensemble [6]. More generally, temperature, chemical environment, sample preparation, and experimental timescales can all affect the relationship between the distribution of biological interest and the one presented to the instrument.

> The possible difference between $$p_{\mathrm{target}}$$ and $$p_{\mathrm{src}}$$ is important, but it is not the problem we will pursue here. We will take the source distribution as the object presented to the experiment and ask what happens next:

Reviewer: ❗️

Hmm this raises the question: then why did you discuss it? And right at the start of the post to boot. I think you need to answer that clearly for the reader. How should they think about this discrepancy between `target` and `src`, and why does it inform an understanding of what comes next?

Reviewer: (don't be too afraid to cut this section if need be)

Author: As I replied in the slack channel:
> I think it’s still important to indicate that figuring out how to recover p_target from p_src is itself an exciting research direction to pursue, and one that very few people have recognized so far (it is possible)
> So I would rather to keep that, some day in the future people might find that interesting.

Reviewer: Right. But if you choose to not defer it to another post, it needs to be tied in somehow. Right now it's a dangling thread that distracts from the main argument that follows.

Why is it important to bring this up *here*? And not later? If you can answer that, I think the change to the text required will become clear.

You can leave stuff out if it doesn't advance the main line of argument. Lawyers have a saying: "trim to win". I am in full agreement.

$$
p_{\mathrm{src}}\longrightarrow\text{measurements}\longrightarrow\text{processed data}.
$$

> Which distinctions within $$p_{\mathrm{src}}$$ survive this chain?

Reviewer: ```suggestion
What happens to $$p_{\mathrm{src}}$$ as it is transformed by these steps?
```

Author: this suggestion is not quite right. p_src remains unchanged, and since it is the quantity we want to recover, it is not transformed by these steps. What changes through these steps is our ability to determine p_src.

Reviewer: point taken, but: then say that!

> ### How experiments constrain the source distribution

Reviewer: ```suggestion
### How experiments project the source distribution
```

Author: I think "constrain" is a better word. In the inverse problem setup, experiments provide additional information that reduces the entropy of the source distribution. In this sense, they act as constraints.

Reviewer: Fair enough. I think it's just challenging for a reader -- who does not have inverse problems loaded in context -- to figure that out when reading through.

Different experiments encode $$p_{\mathrm{src}}$$ in different ways. Broadly, there are two useful cases. In an ensemble-averaged measurement, each observable combines contributions from many molecules. In a single-molecule measurement, individual observations arise from individual molecules, and the distribution appears statistically across many observations. These are not rankings of how much "ensemble information" an experiment contains. They are different ways of constraining the same underlying distribution.

For an ensemble-average measurement, the observable can be written as

$$
\int m(x)\,p_{\mathrm{src}}(x)\,dx + \varepsilon,
$$

where $$m(x)$$ is the forward model for the observable and $$\varepsilon$$ represents measurement uncertainty. The experiment does not reveal the conformation of any individual molecule. Instead, it reports an expectation of the observable over the source distribution.

> Importantly, averaging in **observable space** is not the same as averaging in **conformational space**. Suppose two conformations, $$x_A$$ and $$x_B$$, occur with probabilities $$p_A$$ and $$p_B$$. The measured signal is $$p_A m(x_A)+p_B m(x_B)$$ not, in general, $$m\left(p_A x_A+p_B x_B\right)$$. This distinction matters. If the observable responds differently to the two conformations, their contributions can remain separately constrained even though the measurement averages over many molecules. An X-ray scattering pattern, for example, need not correspond to a fictitious structure halfway between two populated conformations. Spatially distinct states can leave distinct contributions to the measured density. "Averaged" therefore does not mean "only informative about an average structure."

Reviewer: I *do* think the example of altconfs is a familiar and powerful one that could be added here.

Author: You might not believe most junior cryoEM people don't know what is "altconf" and occupancy.

Reviewer: Hmm I do find that surprising! But maybe expected in 3.5 Å land. This comment you can resolve if you like, I will not die on this hill.

For independent observations, their marginal distribution is

$$
\int K(y\mid x,z)\cdot\pi(z)\cdot p_{\mathrm{src}}(x)\,dz\,dx.
$$

Because molecule-to-molecule variation has not been collapsed immediately into a small set of ensemble averages, such data can in principle retain rich information about heterogeneity. But molecule-resolved does not mean that the underlying distribution is directly observed. Noise, unknown nuisance variables, limited sampling, and the dimensionality of conformational variation can make distinct molecular states difficult—or impossible—to distinguish.

The important point is therefore not that one class of experiment "measures ensembles" while the other does not. Both impose constraints on $$p_{\mathrm{src}}$$, but they preserve different distinctions within it. The next question is the one that matters for ensemble inference: **which differences between two possible source distributions would these measurements actually allow us to resolve?**

> ### The question of ensemble resolution

Reviewer: ```suggestion
### Can we measure an "ensemble" resolution?
```

Reviewer: The subtitles also underperform -- think about making some declarative statements. The skimming reader will only see those!

Author: The ensemble resolution is still an open question in my mind. I’m trying to raise the question here rather than propose a solution. If we ask, “Can we measure an ‘ensemble’ resolution?”, would readers assume that we’re going to answer “yes” in what follows, or does it still come across as an open question?

Also, for this blog, I don’t think I’m really trying to cater to skimming readers. I’m a bit tired of optimizing for that—those readers already have plenty to skim through on Twitter every day....

Reviewer: @minhuanli I would just push to say: for _any_ reader, the sub-titles right now don't carry too much information. If you can make them stronger, do so. Having small "headlines" that drive home the points you want to make really helps communicate: both for the distracted audience and the deep reader.
